Sources/Commands/PackageCommands/APIDiff.swift

```diff
@@ -18,8 +18,10 @@ import PackageGraph
 import PackageModel
 import SourceControl
 import SPMBuildCore
+import TSCBasic
```

Comment:
question: What are we using in TSCBasic here?  Can we replace it with a symbol that is not in Swift Tool Support Core?

Reply:
The ByteString type is defined there, so this is just formalizing a dependency that already existed now that a couple extension methods trigger MemberImportVisibility requirements